README.md

```diff
@@ -29,8 +29,22 @@ git clone https://github.com/openMF/web-app.git
 ```
 npm install
 ```
+6. To prevent any CORS error
```

Comment:
You have pushed these changes in https://github.com/openMF/web-app/pull/705 also.

Reply:
Yes, I just realized other changes got added o these too I fixed that sir!